libs/langchain/langchain/vectorstores/milvus.py

```diff
@@ -447,7 +447,9 @@ def add_texts(
             for d in metadatas:
                 for key, value in d.items():
                     if key in self.fields:
-                        insert_dict.setdefault(key, []).append(value)
+                        val = insert_dict.setdefault(key, [])
+                        if isinstance(val, list):
```

Comment:
insert_dict has self._vector_field (`vector` by default) with np.ndarray values. If passed metadata also has a key named `vector`, it will raise error `np.ndarray has no method: append`.

Reply:
```python
        try
            embeddings = self.embedding_func.embed_documents(texts)
        except NotImplementedError:
            embeddings = [self.embedding_func.embed_query(x) for x in texts]
        ...
        insert_dict: dict[str, list] = {
            self._text_field: texts,
            self._vector_field: embeddings,
        }
```

`embeddings` should always be a list since they're generated by `Embeddings` class, no?

Reply:
yes, `embeddings` was inited as a list. but it could be overwritten by user if a key named `self._vector_field` exists in the metadata. 

this is not the only way, another is to limit the keys of metadata to excluding `self._vector_field`.